src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1073,6 +1073,10 @@ private function registerSerializerConfiguration(array $config, ContainerBuilder
         if (isset($config['name_converter']) && $config['name_converter']) {
             $container->getDefinition('serializer.normalizer.object')->replaceArgument(1, new Reference($config['name_converter']));
         }
+
+        if ($container->hasDefinition('property_info')) {
```

Comment:
It can be done directly in the XML file by adding the `on-invalid="ignore"` attribute: https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml#L24